test/e2e/run_dns_test.go

```diff
@@ -99,6 +99,19 @@ var _ = Describe("Podman run dns", func() {
 		Expect(session.OutputToString()).To(ContainSubstring("foobar"))
 	})
 
+	It("podman run domainname sets /etc/hosts and /proc/sys/kernel/domainname", func() {
+		Skip("Waiting for crun release > 1.6")
```

Comment:
crun 1.6 was released some time ago.  Where is the test failing?

Reply:
ah, let me un-comment that and see if this all works...